util/dataframe_ops.py

```diff
@@ -22,14 +22,21 @@ def force_monotonic(dataframe):
 
 
 def compute_rel_diff_dataframe(df1, df2):
-    out = (df1 - df2).abs() / df1.abs()
-    smalls = df1.abs() < CHECK_THRESHOLD
-    out[smalls] = 0.0
+    average = (df1 + df2) / 2
+    out = (df1 - df2) / average
+    out = out.abs()
+    # put 0 if both numbers are very small
+    zeros = (df1.abs() < CHECK_THRESHOLD) & (df2.abs() < CHECK_THRESHOLD)
```

Comment:
I am not sure if this is intentionally, shouldn't it be:
```suggestion
    zeros = (df1.abs() < CHECK_THRESHOLD) and (df2.abs() < CHECK_THRESHOLD)
```

Reply:
Got it,  you are right.